rocky/rocky/views/ooi_view.py

```diff
@@ -93,15 +93,19 @@ def get_queryset(self) -> OOIList:
         return OOIList(
             self.octopoes_api_connector,
             ooi_types=self.get_ooi_types(),
-            valid_time=self.get_observed_at(),
+            valid_time=self.valid_time,
             scan_level=self.get_ooi_scan_levels(),
             scan_profile_type=self.get_ooi_profile_types(),
         )
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         context["mandatory_fields"] = get_mandatory_fields(self.request)
-        context["total_oois"] = len(self.object_list)  # will be auto set on get_queryset()
+        context["observed_at_form"] = self.get_connector_form()
+        context["observed_at"] = self.valid_time
+        context["total_oois"] = len(self.object_list)
+        context["clearance_level_filter_form"] = ClearanceFilterForm(self.request.GET)
+        context["active_filters"] = self.get_active_filters()
```

Comment:
I don't understand why these are added in this PR.

Reply:
I will remove it. this is added 2 times

Reply:
These are moved here to the FilterView because they belong the the filtering values which we only set once. We don't need to set them elsewhere when we inherit this class.

Reply:
This was double sorry, I have removed it